src/Function.cpp

```diff
@@ -574,6 +635,12 @@ Function::OutputForwardDecl(std::ostream &out)
 		int power = rnd_upto(16);
 		out << " __attribute__((aligned(" << (1 << power) << ")))";
 	}
+	vector<FunctionAttribute*>::iterator itr;
+	for(itr = attributes.begin(); itr < attributes.end(); itr++)
+		(*itr)->OutputAttributes(out);
+	if(CGOptions::func_attr_flag())
+		out << "))";
+	flag = 0;
```

Comment:
I need it to be base class member such that I could initialize it here for every function. 

Result - `__attribute__((` for every function.